ptrail/regularization/goemetric_sinuosity.py

```diff
@@ -0,0 +1,188 @@
+"""
+    | Author: Abhishek Gujjar
+"""
+
+import pandas as pd
+from typing import Union
+
+
+def traj_sinuosity_wrapper(trj) -> Union[float, None]:
```

Comment:
To your question of what the trj parameter should be, it should only be either Pandas dataframe or PTRAILDataFrame. Can be typed as Union[pd.DataFrame, PTRAILDataFrame].